final/recommendations/eclipse.tex

```diff
@@ -22,22 +22,22 @@ \subsection{Building languages and projects}
 An obstacle in implementing this feature is the classpath issue as described in
 \cref{sec:classpath}.
 
-\subsection{Back and forth interaction between Eclipse and the REPL}
+\subsection{Interaction between (Spoofax) Eclipse and the Eclipse plugin}
```

Comment:
Should this perhaps just be `Interaction between Spoofax Eclipse and the Eclipse plugin`?